packages/gatsby/src/redux/reducers/status.ts

```diff
@@ -10,7 +14,7 @@ module.exports = (state = { plugins: {} }, action) => {
         PLUGINS_HASH: action.payload,
       }
     case `SET_PLUGIN_STATUS`:
-      if (!action.plugin && !action.plugin.name) {
+      if (!action.plugin || !action.plugin?.name) {
```

Comment:
Looks like this actually fixed a bug based on the snapshot test update! Nice work!

Reply:
🎉 
![](https://media.giphy.com/media/w0MYyUAvYCS64/giphy.gif)